doctor/urls.py

```
@@ -0,0 +1,34 @@
+"""
+URLهای اپلیکیشن Doctor
```

Ruff (0.12.2)

Docstring contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF002)